### PR TITLE
Add dependabot to this repository

### PR DESCRIPTION
**Pull request by alallema**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+- package-ecosystem: pip
+  directory: "/"
+  schedule:
+    interval: "daily"
```
Comment (bidoubiwa) [inline .github/dependabot.yml:6]: In the other repos we use weekly. Did you chose daily on purpose ?

Comment (alallema) [reply .github/dependabot.yml:6]: Yes, I have daily on `digitalocean` repository too, which is the default interval. I think weekly is for repositories that have too much dependabot updates.

**Review by bidoubiwa — APPROVED** [review]
LGTM 🔥

**Review by brunoocasali — APPROVED** [review]

**Comment by alallema** [comment]
bors merge

**Pull request merged**